designsafe/apps/api/decorators.py

```diff
@@ -93,7 +93,7 @@ def decorated_function(request, *args, **kwargs):
 
         jwt_username = jwt_payload.get(settings.AGAVE_JWT_USER_CLAIM_FIELD, '')
         if jwt_username == settings.AGAVE_JWT_SERVICE_ACCOUNT:
-            username = request.GET.get('user', None)
+            username = request.GET.get('user', jwt_username)
```

Comment:
What's the rationale behind setting a default `username` value?

Reply:
With how this if-else is set up, the service account being authenticated with the JWT via request header would not be used to actually make the request, so this change allows that to happen if another user via param `user` is not supplied.

Reply:
Ahh, yes that makes sense. Good catch.

Reply:
I wonder if in this case we'd prefer to not actually "login" the `AGAVE_JWT_SERVICE_ACCOUNT` user, since that user's token isn't actually being used but just verified, and instead skip the login phase if no `user` is supplied in this if statement

Reply:
Going to approve, but curious if anyone has thoughts on this too